src/Message/CancelSubscriptionRequest.php

```diff
@@ -112,6 +113,27 @@ public function getCancelReason()
         return $this->getParameter('cancelReason');
     }
 
+    /**
+     * Set the option to force cancel a subscription.
```

Comment:
Could you expand this comment to explain that it would also dis-entitle the user? Not that we use that feature but someone out there someday might

Reply:
Sure!